catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorKeyboardTest.java

```diff
@@ -110,52 +118,52 @@ public void basicMathOperatorKeysTest() {
 		onView(withId(R.id.formula_editor_keyboard_1)).perform(click());
 		onView(withId(R.id.formula_editor_keyboard_ok)).perform(click());
 
-		onView(withId(R.id.formula_editor_edit_field))
-				.check(matches(withText("( 1 ) + 1 - 1 × 1 ÷ 1 = 1 ")));
+		onView(withId(R.id.formula_editor_edit_field)).check(matches(withText("( 1 ) + 1 - 1 × 1 ÷ 1 = 1 ")));
```

Comment:
i actually do prefer to have the access on the object visually separated from the action on the object to increase readability, espesially with long lines like the above.
please have a look at:
https://confluence.catrob.at/display/CAT/Espresso+testing+-+How+to
Code style guidelines for espresso tests